all-clients/errorcodes.json

```diff
@@ -0,0 +1,235 @@
+    "Mock response maxUsageCount must be greater than 0, %s given",
+    "useLoadableQuery: 'loadQuery' should not be called during render. To start a query during render, use the 'useBackgroundQuery' hook.",
+    "Unknown query %o requested in refetchQueries options.include array"
+  ],
+  "allConditions": [
+    null,
+    "typeof dataId === \"string\"",
+    "extracted !== void 0",
+    "!old || old === which",
+    null,
+    "!isReference(value)",
+    "options.fetchPolicy !== 'cache-and-network'",
+    "Array.isArray(cacheKeys)",
+    "fragment",
+    "pollInterval",
+    "mutation",
+    "fetchPolicy === 'network-only' ||\n    fetchPolicy === 'no-cache'",
+    "options.query",
+    "options.query.kind === 'Document'",
+    "!options.returnPartialData",
+    "!options.pollInterval",
+    "options && (typeof options.sha256 === 'function' ||\n    typeof options.generateHash === 'function')",
+    "forward",
+    "context && context.client",
+    "'createContext' in React",
+    "context.client",
+    "this.withRef",
+    "operationOptions.withRef",
+    "!!client",
+    "queryRef.promiseCache",
+    "suspenseCache",
+    "supportedFetchPolicies.includes(fetchPolicy)",
+    "!!document && !!document.kind",
+    "!fragments.length ||\n    (queries.length || mutations.length || subscriptions.length)",
+    "queries.length + mutations.length + subscriptions.length <= 1",
+    "definitions.length === 1",
+    "operation.type === type",
+    "queryWithoutConnection",
+    "evaledValue !== void 0",
+    "directiveArguments && directiveArguments.length === 1",
+    "ifArgument.name && ifArgument.name.value === 'if'",
+    "ifValue &&\n    (ifValue.kind === 'Variable' || ifValue.kind === 'BooleanValue')",
+    "fragments.length === 1",
+    "doc && doc.kind === 'Document'",
+    "operations.length <= 1",
+    "queryDef && queryDef.operation === 'query'",
+    "doc.kind === 'Document'",
+    "doc.definitions.length <= 1",
+    "fragmentDef.kind === 'FragmentDefinition'",
+    "internalQueryRef.promiseCache",
+    "\"createContext\" in React",
+    "!fragments.length ||\n    queries.length ||\n    mutations.length ||\n    subscriptions.length",
+    "options.fetchPolicy !== \"cache-and-network\"",
+    "fetchPolicy === \"network-only\" || fetchPolicy === \"no-cache\"",
+    "options.query.kind === \"Document\"",
+    "options &&\n    (typeof options.sha256 === \"function\" ||\n        typeof options.generateHash === \"function\")",
+    "ifArgument.name && ifArgument.name.value === \"if\"",
+    "ifValue &&\n    (ifValue.kind === \"Variable\" || ifValue.kind === \"BooleanValue\")",
+    "doc && doc.kind === \"Document\"",
+    "queryDef && queryDef.operation === \"query\"",
+    "doc.kind === \"Document\"",
+    "fragmentDef.kind === \"FragmentDefinition\"",
+    "mockedResponse.maxUsageCount > 0",
+    "!calledDuringRender()"
+  ],
+  "allFiles": [
+    null,
+    "@apollo/client/cache/inmemory/entityStore.js",
```

Comment:
Deliberately skipping the 0th index here so our byte arrays don't contain zeroes, which can help with "at a glance" detection of "obviously wrong" elements.